app/models/application_letter.rb

```diff
@@ -46,7 +55,21 @@ def after_deadline?
   # @param none
   # @return [Boolean] true if status changes are allowed
   def status_change_allowed?
```

Comment:
This functions is used for validations, and we never tested our validation functions afaik.
Also, there are model tests in this PR which cover that (via validations) :)